src/yuzu/main.cpp

```diff
@@ -2189,10 +2193,19 @@ static bool IsSingleFileDropEvent(QDropEvent* event) {
 }
 
 void GMainWindow::dropEvent(QDropEvent* event) {
-    if (IsSingleFileDropEvent(event) && ConfirmChangeGame()) {
-        const QMimeData* mimeData = event->mimeData();
-        QString filename = mimeData->urls().at(0).toLocalFile();
-        BootGame(filename);
+    if (!IsSingleFileDropEvent(event)) {
+        return;
+    }
+
+    const QMimeData* mime_data = event->mimeData();
+    const QString& filename = mime_data->urls().at(0).toLocalFile();
```

Comment:
```suggestion
    const QString filename = mime_data->urls().at(0).toLocalFile();
```

toLocalFile() returns by value, not by reference.